YACS-RCOS/yacs.n
# Filtered out courses with no sections (in course list)

**Issue**

> closes #220 

Filtered out courses from the loaded CSV that do not contain any sections.

**Database Changes/Migrations**

n/a

**Test Procedure**

Show procedure to test functionality with a clear procedure

 1. Load Spring 2020 CSV
 2. Notice courses on course list
 3. Ensure all courses in list have sections (show on schedule w/ a time)
